microsoft/kiota
# Feat: hand over generated plugin manifest

Sends the plugin manifest to TTK if source of generation is TTK.

## How to test

- Follow steps specified in https://github.com/microsoft/kiota/blob/main/vscode/microsoft-kiota/debugging.md#running-in-debug-mode
- On pressing F5 to open a new vscode debug window, press the key combination ctrl + shift + p to open the command pallete.
- Select the Debug: Open Link command which will provide an input box to paste your url. e.g. "vscode://ms-graph.kiota/OpenDescription?descriptionUrl%3Dhttps%3A%2F%2Fraw.githubusercontent.com%2Fgithub%2Frest-api-description%2Fmain%2Fdescriptions%2Fghes-3.0%2Fghes-3.0.json&kind=Plugin&type=ApiPlugin&name=GitHubClientPlugin&source=ttk"
- Follow the prompts to open the link in vscode debug window.
- When tree loads in the side panel, select the nodes you want to include in your plugin, and select the "generate" button at the top of the tree structure.
- The generate process should skip the questions with provided answers in the deeplink params.
- Once generation is done, a new set of prompts appear powered by ttk and you will generate a teams app
